cmd/brew-upgrade-mysql

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -e
+
+update_my_cnf() {
+  CNF_PATH="$(brew --prefix)/etc/my.cnf"
+
+  echo "Backing up old my.cnf file..."
+  cp -vf $CNF_PATH "$CNF_PATH.github"
+
+  cat > $CNF_PATH <<-EOM
+# For advice on how to change settings please see
+# http://dev.mysql.com/doc/refman/5.7/en/server-configuration-defaults.html
+
+[mysqld]
+
+innodb_strict_mode=OFF
+optimizer_switch='index_merge_intersection=OFF'
+query_cache_size=0
+sql_mode=NO_ENGINE_SUBSTITUTION
+EOM
+}
+
+set +e
+brew ls mysql@5.6 &>/dev/null
+MYSQL_5_6="$?"
+
+brew ls mysql@5.7 &>/dev/null
+MYSQL_5_7="$?"
+set -e
+
+if [[ $MYSQL_5_6 == "0" || $MYSQL_5_7 == "0" ]]; then
+  if [[ $MYSQL_5_6 == "0" ]]; then
+    set +e
+    brew services stop mysql@5.6
+    set -e
+
+    brew uninstall mysql@5.6
+    brew install mysql@5.7
+  fi
+
+  if [[ $MYSQL_5_7 == "0" ]]; then
+    set +e
+    brew services stop mysql@5.7
+    set -e
+  fi
+
+  update_my_cnf
+
+  # Upgrade the MySQL tables
+  echo "Upgrading MySQL Tables ..."
+  brew services start mysql@5.7
+
+  while ! nc -z localhost 3306; do
+    echo "Waiting for MySQL to be available..."
+    sleep 2
+  done
+
+  $(brew --prefix mysql@5.7)/bin/mysql_upgrade -u root
+  brew services stop mysql@5.7
+  brew services start mysql@5.7
+fi
+
+EXPECTED_EXIT="1"
```

Comment:
Is this variable used anywhere?

Reply:
Nope, just copy paste from the other files.